src/main/kotlin/net/maizegenetics/phgv2/pathing/BuildKmerIndex.kt

```diff
@@ -65,15 +62,17 @@ class BuildKmerIndex: CliktCommand(help="Create a kmer index for a HaplotypeGrap
         .default(1)
 
     val hvcfDir by option("--hvcf-dir", help = "Path to directory holding hVCF files. Data will be pulled directly from these files instead of querying TileDB")
-        .default("")
+        .required()//Todo: make this optional by adding .default("")
+
+
 
     override fun run() {
         //either tiledbPath or agcPath must be provided
 
 
         //build the haplotypeGraph
         val graph = buildHaplotypeGraph()
-        val hashToHapidMap = processGraphKmers(graph, agcPath, maxHaplotypeProportion,  hashMask, hashFilterValue)
+        val hashToHapidMap = processGraphKmers(graph, "${dbPath}", maxHaplotypeProportion,  hashMask, hashFilterValue)
```

Comment:
just dbPath seems cleaner than "${dbPath}", though they resolve to the same thing.

Reply:
Yeah I originally had it be "${dbPath}/assemblies.agc" but then through testing realized that the file name is not needed just the path.  I will remove the curly brackets.